components/kyma-environment-broker/common/hyperscaler/azure/config.go

```diff
@@ -1,10 +1,16 @@
 package azure
 
 import (
+	"github.com/Azure/azure-sdk-for-go/services/containerregistry/mgmt/2019-05-01/containerregistry"
 	"github.com/Azure/go-autorest/autorest/azure"
 	"github.com/pkg/errors"
 )
 
+type StepConfig struct {
```

Comment:
I think the naming is not good. What the `Disabled` field mean? Azure is disabled or container registry in Azure is disabled?

Reply:
I agree, do you think renaming `Disabled` field to `ContainerRegistryDisabled` would do the trick?